mmdet3d/evaluation/functional/panoptic_seg_eval.py

```diff
@@ -0,0 +1,329 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from mmengine.logging import print_log
+
+
+class EvalPanoptic:
```

Comment:
You can refer to [InstanceSegMetric](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/mmdet3d/evaluation/metrics/instance_seg_metric.py) to implement the PanopticSegMetric in `mmdet3d/evaluation/metrics/panoptic_seg_metric.py` and some evaluation utils can be written in `mmdet3d/evaluation/functional/panoptic_seg_eval.py`

Reply:
OK, I've added type hints and a function wrapper for this util.